package.json

```diff
@@ -94,11 +94,12 @@
     ]
   },
   "resolutions": {
+    "js-yaml": "4.1.1",
     "react": "^17",
     "react-docgen-typescript": "2.2.2"
   },
   "devDependencies": {
-    "@codemod/cli": "2.3.2",
+    "@codemod/cli": "3.3.0",
```

Comment:
Do we use codemod at all? Maybe we can remove it?
CC @doniyor2109

Reply:
We are not using it recently. I think we can remove it